src/datasources/bridge-api/lifi-api.service.ts

```diff
@@ -0,0 +1,179 @@
+import type { HttpErrorFactory } from '@/datasources/errors/http-error-factory';
```

Comment:
This negates needing to use the SDK. We should remove the methods which are not needed and reduce complexity of those which are. On top of that, we should only validate the properties which are required for mapping.